src/quo2/components/wallet/account_card/view.cljs

```diff
@@ -0,0 +1,70 @@
+(ns quo2.components.wallet.account-card.view
+  (:require [react-native.core :as rn]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.icon :as icon]
+            [quo2.components.wallet.account-card.style :as style]
+            [quo2.components.buttons.button.view :as button]
+            [utils.i18n :as i18n]
+            [quo2.components.markdown.text :as text]))
+
+(defn user-account-view
```

Comment:
Usually we don't append `view` or `component` suffixes unless there's a naming conflict and the function name would shadow a parameter. So just `user-account` is enough.